cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractOperator.java

```diff
@@ -95,11 +90,11 @@ public abstract class AbstractOperator<
      * OperatorMetricsHolder instance. This constructor is used by subclasses which want to use specialized metrics
      * ¨holder such as the subclasses dealing with KafkaConnector resources and their metrics.
      *
-     * @param vertx             Vert.x instance
-     * @param kind              Resource kind which will be operated by this operator
-     * @param resourceOperator  Resource operator for given custom resource
-     * @param metrics           MetricsHolder for managing operator metrics
-     * @param selectorLabels    Selector labels for selecting custom resources which should be operated
+     * @param vertx            Vert.x instance
+     * @param kind             Resource kind which will be operated by this operator
+     * @param resourceOperator Resource operator for given custom resource
+     * @param metrics          MetricsHolder for managing operator metrics
+     * @param selectorLabels   Selector labels for selecting custom resources which should be operated
```

Comment:
Can we avoid all the formatting changes here and in the other places?

Reply:
Sure, its my IDE doing it :) Will revert all of it